### PR TITLE
docs: Update UI Text

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### What type of PR is this?

/kind documentation

### What this PR does / why we need it:
Update UI text.

### Which issue(s) this PR fixes:

Fixes ## [4076](https://github.com/kubesphere/kubesphere/issues/4076)

### Special notes for reviewers:
```
Code changes made to the following files:

src/components/Forms/GrayRelease/Components/index.jsx
src/components/Forms/GrayRelease/Components/Card/index.jsx
src/components/Forms/GrayRelease/Version/index.jsx
src/components/Forms/GrayRelease/Version/ContainerList/Card.jsx
src/components/Forms/GrayRelease/PolicyConfig/Bluegreen/VersionSelect/index.jsx
src/components/Forms/ImageBuilder/B2IForm/index.jsx
src/components/Forms/ImageBuilder/B2IForm/BinaryFileUploader/index.jsx
src/components/Forms/ImageBuilder/S2IForm/index.jsx
src/components/Modals/GrafanaModal/Uploader/index.jsx
src/pages/projects/containers/GrayRelease/Jobs/index.jsx
src/pages/projects/containers/GrayRelease/Jobs/Item.jsx
```

### Does this PR introduced a user-facing change?

```release-note
None
```

/assign @harrisonliu5 
Please review this PR, thanks.

### CODE REVIEW

**Comment by Felixnoo** [comment]
@Patrick-LuoYu 
This PR only had few conflicts with your PR #2255 and I resolved them. FYI
Xnip2021-09-18_11-22-46
.

File: src/components/Forms/GrayRelease/Version/index.jsx [hunk 1]
```diff
@@ -70,11 +70,7 @@ export default class Version extends ContainerSettings {
       .then(resp => {
         if (resp.exist) {
           return callback({
-            message: t(
-              `${t('DEPLOYMENTS')} ${name} ${t('exists')}, ${t(
-                'version number is invalid'
-              )}`
-            ),
+            message: t('GRAY_DEPLOY_VERSION_TIP', { name: t(name) }),
```
Comment (harrisonliu5) [inline src/components/Forms/GrayRelease/Version/index.jsx:73]: is it right `t(name)`?

Comment (Felixnoo) [reply src/components/Forms/GrayRelease/Version/index.jsx]: Thanks for your comment. Fixed in [this commit](https://github.com/kubesphere/console/pull/2264/commits/43711be7c0773eccbfdf7c8f928518686454580c#diff-717636810ecd663d084d50c11ce59b5d79e149eacdd4869abbfc6a7e0b7f9612).

**Pull request merged**